controllers/ibmvpccluster_controller.go

```diff
@@ -152,7 +155,50 @@ func (r *IBMVPCClusterReconciler) reconcile(clusterScope *scope.ClusterScope) (c
 		}
 	}
 
-	clusterScope.IBMVPCCluster.Status.Ready = true
+	if clusterScope.IBMVPCCluster.Spec.ControlPlaneLoadBalancer != nil {
+		loadBalancer, err := r.getOrCreate(clusterScope)
+		if err != nil {
+			return ctrl.Result{}, errors.Wrapf(err, "failed to reconcile Control Plane LoadBalancer for IBMVPCCluster %s/%s", clusterScope.IBMVPCCluster.Namespace, clusterScope.IBMVPCCluster.Name)
+		}
+
+		if loadBalancer != nil {
+			clusterScope.SetLoadBalancerID(loadBalancer.ID)
+			clusterScope.Info("LoadBalancerID - " + *clusterScope.GetLoadBalancerID())
+			clusterScope.SetLoadBalancerState(*loadBalancer.ProvisioningStatus)
+			clusterScope.Info("LoadBalancerState - " + string(clusterScope.GetLoadBalancerState()))
```

Comment:
This looks to be like a debugging message, lets bump the verbose level to avoid printing in the default

Reply:
Fixed.